tools/src/tensil/tools/compiler/TfFrontend.scala

```diff
@@ -124,8 +124,13 @@ class TfFrontend(
         )
   }
 
-  def mkConstsDimensions(shape: Shape): MemoryDimensions =
-    ConstsDimensions(shape)
+  def mkConstsDimensions(shape: Shape, transpose: Boolean): MemoryDimensions =
+    if (transpose)
+      throw new CompilerException(
+        s"Transposing consts is not supported"
```

Comment:
Why is that? Or is it just that TF consts will never set `transpose = true`?

Reply:
Correct.